omod/src/main/webapp/fragments/patientdashboard/contactInfoInline.gsp

```diff
@@ -4,12 +4,12 @@
 
 <div class="contact-info-inline">
     <span>
-        ${ ui.format(config.patient.personAddress).replace("\n", ", ")}
+        ${ ui.encodeHtmlContent(ui.format(config.patient.personAddress).replace("\n", ", "))}
         <em>${ ui.message("coreapps.person.address")}</em>
     </span>
     <span class="left-margin">
         <span id="coreapps-telephoneNumber">
-            ${config.patient.telephoneNumber ?: ''}
+            ${ ui.encodeHtmlContent(config.patient.telephoneNumber ?: '') }
```

Comment:
Awesome, thanks for merging my first ever pull request!

I think RA-649 can be resolved after this merge.